master/buildbot/status/web/hooks/github.py

```diff
@@ -168,6 +184,73 @@ def _process_change(self, payload, user, repo, repo_url, project):
 
         return changes
 
+    def _process_pull_request(self, payload, user, repo, repo_url, project, codebase=None):
```

Comment:
Since it's a private method, why `codebase` parameter is needed at all?  (It's available as `self._codebase`)